### PR TITLE
CI updates

### PR DESCRIPTION
**Pull request by rbberger**
## PR Summary

Future proof our re-git CI and some cosmetic changes.

## PR Checklist

- [ ] Adds a test for any bugs fixed. Adds tests for new features.
- [ ] Format your changes by using the `make format` command after configuring with `cmake`.
- [ ] Document any new features, update documentation for changes made.
- [ ] Make sure the copyright notice on any files you modified is up to date.
- [ ] After creating a pull request, note it in the CHANGELOG.md file
- [ ] If preparing for a new release, update the version in cmake.

### CODE REVIEW

**Review by Yurlungur — APPROVED** [review]
Nice cleanup

**Comment by Yurlungur** [comment]
@rbberger feel free to merge this when you're ready. Probably worth double checking no syntax errors on the Darwin CI if you haven't already done so

**Comment by rbberger** [comment]
@Yurlungur Kudos really need to go to @opensdh who proposed the `section()` change for [flecsi](https://github.com/flecsi/flecsi).

Pushed both the spiner and singularity-eos branches to re-git. Once they all pass, I'll merge.

**Pull request merged**
